### PR TITLE
Migrate to using Go Module to manage dependencies

### PR DESCRIPTION
**Pull request by huydoan2**
Besides using Go Module (adding go.mod and go.sum), there are several changes:

1. Upgraded Go version to 1.14 alongside with other packages.
2. Modified unit tests to accommodate changes.

### CODE REVIEW

**Comment by mrutkows** [comment]
Appreciate your work on this Huy... hopefully, this a will address the long-standing desire to use go mod as referenced in this issue: https://github.com/apache/openwhisk-wskdeploy/issues/1082

File: tests/dat/manifest_data_compose_actions_for_limits.yaml [hunk 1]
```diff
@@ -21,7 +21,6 @@ packages:
       hello1:
         function: ../src/integration/helloworld/actions/hello.js
         limits:
-          timeout: 1
           timeout: 600000
```
Comment (rabbah) [inline tests/dat/manifest_data_compose_actions_for_limits.yaml:24]: Should this be 60000 ?

Comment (huydoan2) [reply tests/dat/manifest_data_compose_actions_for_limits.yaml:24]: Not sure. I didn't change that value. Additionally, that test file is used in parsing tests so the values may not matter much.

Comment (mrutkows) [reply tests/dat/manifest_data_compose_actions_for_limits.yaml:24]: The value was not intended to be 60 seconds... just testing input matches output against some larger (max) value. Here 10 minutes...

**Comment by mrutkows** [comment]
@style95 Hi Dom, could you and colleagues (who recently showed wskdeploy IDE plugins) please review so we make sure we are not going to break you if we move to merge.  Thanks!

**Comment by upgle** [comment]
Thank you for your contribution 🎉 

Could you please update the README.md? Some contents need to be modified for `go mod`.
https://github.com/apache/openwhisk-wskdeploy#developers-should-use-go-deps-and-go-build-not-go-get

And I've checked there is no issue for VSCode openwhisk plugin.
@KeonHee Could you check it for Jetbrains extension?

**Comment by KeonHee** [comment]
@mrutkows @upgle Thanks for noti & @huydoan2 Thanks for your contribution :)

It works well with Jetbrain plugin too.

**Comment by huydoan2** [comment]
Thank you, all.

@upgle We will need to clean up the doc and the Makefile in a separate PR :).

**Pull request merged**